code/game/area/areas.dm

```diff
@@ -55,7 +55,7 @@
 	)
 	var/list/forced_ambience = null
 	var/sound_env = STANDARD_STATION
-	var/turf/base_turf//The base turf type of the area, which can be used to override the z-level's base turf

```

Comment:
ree, this was grammatically correct

Reply:
Oh, right. Sorry. Had to miss the one apostrophe. I really only meant to put in the space.